lib/merge.js

```diff
@@ -160,7 +160,10 @@ function getCommitMessage(mergeCommitMessage, pullRequest) {
   } else if (mergeCommitMessage === "pull-request-title-and-description") {
     return pullRequest.title + "\n\n" + pullRequest.body;
   } else {
-    throw new Error(`unknown commit message value: ${mergeCommitMessage}`);
+    ["number", "title", "body"].forEach(prProp => {
+      mergeCommitMessage.replace(new RegExp(`{pullRequest.${prProp}}`, 'g'), pullRequest[prProp]);
```

Comment:
I just noticed this won't work, as `replace` doesn't change the string, the return value needs to be used!

Reply:
oops, you're right, my junior fault :grin:

Reply:
https://github.com/pascalgn/automerge-action/pull/45/commits/069a0ef207c6e3e387f2022d63fe6151765d54cd